front/app/api/graph_data_units/useGraphDataUnits.ts

```diff
@@ -16,7 +16,7 @@ type Props = {
 
 const useGraphDataUnits = <Response extends BaseResponseData>(
   parameters: ParametersLive,
-  { enabled, onSuccess }: Props = { enabled: true }
+  { enabled = true, onSuccess }: Props = { enabled: true }
```

Comment:
Without it, `enabled` is `undefined` when only `onSuccess` is provided. `undefined` is passed into `useGraphDataUnitsPublished` there it becomes `true`, which causes `web_api/v1/reports/graph_data_units/published` requests in Report Builder on BO.

Reply:
Ha, I also found this issue on my branch: https://github.com/CitizenLabDotCo/citizenlab/pull/6838/commits/13fdd926d7dfadbb942949cb3050ba831e8c9bdf